src/pages/landing-page/landing-page.css

```diff
@@ -0,0 +1,79 @@
+.landing-img{
+    width:100%;/*will change later*/
+    margin-bottom: 5rem;/*will change later*/
+}
+.landing-img img{
+    height: 100%;
+    width: 100%;
+    object-fit: cover;
+}
+
+/* category */
+.category{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    padding-bottom: 1.4rem;
+}
+.category-heading{
+    font-size: 1.8rem;
+    letter-spacing: 0.1rem;
+    padding-bottom: 1rem;
+}
+.category-heading span{
+    color: var(--color-primary);
+}
+
+.category-images{
+    width: 90%;
+    display: flex;
+    flex-direction: row;
```

Comment:
just a suggestion ,flex  has a default value flex-direction : row , you can remove it

Reply:
Yes, I did the changes